livekit/src/rtc_engine/rtc_session.rs

```diff
@@ -747,6 +765,21 @@ impl SessionInner {
                     }
                 }
             }
+            RtcEvent::DataChannelBufferedAmountChange { sent: _, amount, kind } => {
+                match kind {
+                    DataPacketKind::Lossy => {
+                        // Do nothing at this moment
```

Comment:
I agree with your assessment that this issue isn't as pressing on the lossy channel, but one thing we'd want to avoid is that the data channel just errors out if the buffered_amount surpassing an internal maximum value. I think @theomonnom mentioned that right now it will raise an error in this case? 
For that reason I think it could also make sense to include the same logic for the lossy data channel

Reply:
Yes, I think we should use the same logic to avoid internal RtcError